# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: widget/gtk/DMABufLibWrapper.cpp
# Commit: efdd313b337a
# Full Hash: efdd313b337a74a3dfbb788c8a969d9136c7036c
# Author: Stanca Serban <sstanca@mozilla.com>
# Date: 2025-02-05 16:56:59
# Regressor Bug: 1944490
# File Overlap Count: 1
# Description:
#   Bug 1944490 - Fix includes now we use DMABufFormats on X11 as well. CLOSED TREE
# ==============================================================================

diff -r 8f8fbd5236fa -r efdd313b337a widget/gtk/DMABufLibWrapper.cpp
--- a/widget/gtk/DMABufLibWrapper.cpp	Wed Feb 05 12:23:45 2025 +0000
+++ b/widget/gtk/DMABufLibWrapper.cpp	Wed Feb 05 14:41:44 2025 +0200
@@ -6,9 +6,9 @@
  * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
 
 #include "DMABufLibWrapper.h"
+#  include "DMABufFormats.h"
 #ifdef MOZ_WAYLAND
 #  include "nsWaylandDisplay.h"
-#  include "DMABufFormats.h"
 #endif
 #include "base/message_loop.h"    // for MessageLoop
 #include "mozilla/gfx/Logging.h"  // for gfxCriticalNote
